tests/components/blue_current/test_sensor.py

```diff
@@ -85,69 +84,106 @@
 }
 
 
+async def test_sensors_created(hass: HomeAssistant) -> None:
+    """Test if all sensors are created."""
+    await init_integration(
+        hass,
+        "sensor",
+        charge_point,
+        charge_point_status | charge_point_status_timestamps,
+        grid,
+    )
+
+    entity_registry = er.async_get(hass)
+
+    sensors = er.async_entries_for_config_entry(entity_registry, "uuid")
+    assert len(charge_point_status.keys()) + len(
```

Comment:
We can remove `.keys()` when taking the length of a dict.
```suggestion
    assert len(charge_point_status) + len(
```